pokemon-showdown

```diff
@@ -222,13 +189,13 @@ if (!process.argv[2] || /^[0-9]+$/.test(process.argv[2])) {
 	case 'json-team':
 		{
 			sucraseBuild();
-			var Teams = require('./.sim-dist/teams').Teams;
-			var Streams = require('./.lib-dist/streams');
+			var Teams = require('./.sim/teams').Teams;
+			var Streams = require('./.lib/streams');
```

Comment:
This . error seems common, are you doing this by hand? Next time `perl -pi -e "s/.lib-dist/lib/g" **/*.ts` etc. I'm stopping the review here, but I may be able to look at it again after you double check your work.

Reply:
Nope, did it with VSCode find and replace. Probably just missed a . in the replace. One sec.

Reply:
Reverted and ran the proper find/replace this time.